### ISSUE TITLE
please add 3.21.91 to shell-version

### ISSUE THREAD

**Issue by pubcoder** [phocean/TopIcons-plus#36]
I just upgraded gnome-shell to 3.21.91 to witness that TopIcons-plus is gone.
The workaround is to add "3.21.91" to "shell-version" in metadata.json.
thanks!

**Comment by phocean** [comment]
I have already done that and regularly receive this kind of "bug" report. And again, this is not my fault if Gnome extension website is taking a loooooooot of time to validate the updated versions that I submit there.

Anyway this will be irrelevant in Gnome 3.22 as they will disable version check by default:
https://git.gnome.org/browse/gnome-shell/commit/?id=5e0e3ed

There is a command for that:

`gsettings set org.gnome.shell disable-extension-version-validation "true"`

Hope this help.

**Comment by mikucionisaau** [comment]
1) I am not using Gnome extension website, I cloned this repository and typed "make".
2) There is only "3.21" among versions and "3.21.91" is not there, so it is not "fixed", i.e. I really had to add "3.21.91" in order to get it working. So it seems the version check got even worse...
3) it's not your fault, I am also fed up with this version check (I had to revisit three other extensions). Thanks for the command to disable the check, I really appreciate it.

**Comment by pubcoder** [comment]
1) I am not using Gnome extension website, I cloned this repository and typed "make".
2) There is only "3.21" among versions and "3.21.91" is not there, so it is not "fixed", i.e. I really had to add "3.21.91" in order to get it working. So it seems the version check got even worse...
3) it's not your fault, I am also fed up with this version check (I had to revisit three other extensions). Thanks for the command to disable the check, I really appreciate it.
